src/service.ts

```diff
@@ -722,7 +744,8 @@ export class GrpcService extends Service {
    * @param {object} response - The grpc response.
    * @return {object|null}
    */
-  private static decorateGrpcResponse_(obj, response) {
+  // tslint:disable-next-line no-any
+  private static decorateGrpcResponse_(obj: {}, response: any) {
```

Comment:
Can response be `GrpcResponse`?

Reply:
No :( Sadly the types in here are a bit of a hairball at the moment.  After this lands, I want to go through with the debugger, and dump objects to actually understand what's happening.  This is loose right now to get us past https://github.com/googleapis/nodejs-logging/issues/265.